knip.jsonc

```diff
@@ -32,14 +32,17 @@
                 "c8",
 
                 // These will be removed in https://github.com/eslint/eslint/pull/18011
-                "eslint-plugin-eslint-comments",
+                "@eslint-community/eslint-plugin-eslint-comments",
                 "eslint-plugin-jsdoc",
                 "eslint-plugin-n",
                 "eslint-plugin-unicorn",
 
                 // Ignore until Knip has a wdio plugin:
                 "@wdio/*",
-                "rollup-plugin-node-polyfills"
+                "rollup-plugin-node-polyfills",
+
+                // FIXME: not sure why is eslint-config-eslint reported as unused
+                "eslint-config-eslint"
```

Comment:
it can be fixed by putting `eslint.config.js` in the entry.

Reply:
> it can be fixed by putting `eslint.config.js` in the entry.

I tried that first, in https://github.com/eslint/eslint/pull/18400/commits/9f0f9b65e3ad892d46b8610eec22de693232ab62, but it didn't work. Not sure why.

Reply:
> and L35~38 can also be removed. :)

Done in https://github.com/eslint/eslint/pull/18400/commits/f67403a0b4c6fb8f484f0289c10a3daa576133d9